cmd/ui/src/views/DatabaseManagement/DatabaseManagement.tsx

```diff
@@ -0,0 +1,307 @@
+        }
+        case 'selection': {
+            const { targetName, checked } = action;
+            return {
+                ...state,
+                [targetName]: checked,
+                noSelectionError: false,
+            };
+        }
+        case 'open_dialog': {
+            const { collectedGraphData, dataQualityHistory, fileIngestHistory, highValueSelectors } = state;
+            if (
+                [collectedGraphData, dataQualityHistory, fileIngestHistory, highValueSelectors].filter(Boolean)
+                    .length === 0
+            ) {
+                return {
+                    ...state,
+                    noSelectionError: true,
+                };
+            } else {
+                return {
+                    ...state,
+                    noSelectionError: false,
+                    openDialog: true,
+                };
+            }
+        }
+        case 'close_dialog': {
+            return {
+                ...state,
+                openDialog: false,
+            };
+        }
+        default: {
+            return state;
+        }
+    }
+};
+
+const useDatabaseManagement = () => {
+    const [state, dispatch] = useReducer(reducer, initialState);
+    const { collectedGraphData, highValueSelectors, fileIngestHistory, dataQualityHistory } = state;
+
+    const tierZeroAssetGroupId = useSelector(selectTierZeroAssetGroupId);
+
+    const mutation = useMutation({
+        mutationFn: async ({ deleteThisData, assetGroupId }: { deleteThisData: DataTypes; assetGroupId: number }) => {
+            return apiClient.databaseManagement({
+                ...deleteThisData,
+                assetGroupId,
+            });
+        },
+        onError: () => {
+            // show UI message that data deletion failed
+            dispatch({ type: 'mutation_error' });
+        },
+        onSuccess: () => {
+            // show UI message that data deletion is happening
+            dispatch({ type: 'mutation_success' });
+        },
+    });
+
+    const handleMutation = () => {
+        mutation.mutate({
+            deleteThisData: {
+                collectedGraphData,
+                dataQualityHistory,
+                fileIngestHistory,
+                highValueSelectors,
+            },
+            assetGroupId: tierZeroAssetGroupId,
+        });
+    };
+
+    return { handleMutation, state, dispatch };
+};
+
+const DatabaseManagement = () => {
+    const theme = useTheme();
+
+    const { handleMutation, state, dispatch } = useDatabaseManagement();
+
+    const { collectedGraphData, highValueSelectors, fileIngestHistory, dataQualityHistory } = state;
+
+    const handleCheckbox = (event: React.ChangeEvent<HTMLInputElement>) => {
+        dispatch({
+            type: 'selection',
+            targetName: event.target.name,
+            checked: event.target.checked,
+        });
+    };
+
+    return (
+        <ContentPage title='Clear BloodHound data'>
+            <div>
```

Comment:
It would be nice to swap this out for an MUI \<Box/> so that it applies the generated MUI css classes. It is effectively the same right now but if we ever start introducing any theming (like dark mode 👀 ) it will be easier to have MUI handle sweeping style changes rather than having to update this div manually. Wherever possible, try to use MUI components.